LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020-2021 The SymbiFlow Authors
```

Comment:
You never update this file. Please revert this change.

Reply:
ok

Reply:
done